src/routes/(admin)/tests/test-[type]/[id]/+page.svelte

```
@@ -51,6 +52,16 @@
 		}
 	});
 
+	$effect(() => {
+		if (convertTemplate && currentScreen === typeOfScreen.primary) {
+			if (page.url.searchParams.has('template_id')) {
+				const url = new URL(page.url);
+				url.searchParams.delete('template_id');
+				goto(url, { invalidateAll: true, noScroll: true, keepFocus: true });
```

ESLint

[error] Unexpected goto() call without resolve().

(svelte/no-navigation-without-resolve)